README.md

```diff
@@ -218,6 +218,22 @@ Currently, the following cases are available for this task:
 - Go
   - `plain/plain.go`: An empty function that does nothing.
 
+### Task: Code Repair
+
+Code repair is the task of repairing source code with compilation errors.
+
+For this task, we introduced the `mistakes` repository, which includes five examples of source code with compilation errors for both Java and Go.
```

Comment:
Would remove this sentence cause the number of examples and languages could change.